content/es/devops.md

```diff
@@ -0,0 +1,16 @@
+---
+title: DevOps
+status: Completed
+category: concept
+---
+
+## Qué es
+DevOps es una metodología en la que los equipos son dueños de todo el proceso, desde el desarrollo de la aplicación hasta las operaciones en producción, de ahí el término DevOps. Va más allá de implementar un conjunto de tecnologías y requiere un cambio completo en la cultura y los procesos. DevOps requiere grupos de ingenieros que trabajen en componentes pequeños (en lugar de una función completa), reduciendo las transferencias, una fuente común de errores.
+
+## Problema que aborda
+Tradicionalmente, en organizaciones complejas con [aplicaciones monolíticas](/monolithic_apps/) e con arquitecturas con [alto acoplamiento](/tightly_coupled_architectures/), el trabajo generalmente se distribuía entre varios equipos. Esto dio lugar a numerosos traspasos y largos plazos de entrega. Cada vez que un componente o alguna actualización estaba lista, se colocaba en una fila para el siguiente equipo. Debido a que las personas solo trabajaron en una pequeña parte del proyecto, este enfoque condujo a una falta de propiedad. Su objetivo era entregar el trabajo al siguiente grupo, en vez de ofrecer la funcionalidad correcta al cliente - un claro desajuste de prioridades.
```

Comment:
```suggestion
Tradicionalmente, en organizaciones complejas con [aplicaciones monolíticas](/monolithic_apps/) y arquitecturas [altamente acopladas](/tightly_coupled_architectures/), el trabajo generalmente se distribuía entre varios equipos. Esto dio lugar a numerosos traspasos y largos plazos de entrega. Cada vez que un componente o alguna actualización estaba lista, se colocaba en una fila para el siguiente equipo. Debido a que las personas solo trabajaron en una pequeña parte del proyecto, este enfoque condujo a una falta de propiedad. Su objetivo era entregar el trabajo al siguiente grupo, en vez de ofrecer la funcionalidad correcta al cliente - un claro desajuste de prioridades.
```

Refraseo un poco para evitar los 3 con seguidos: `con` apps.. `con` arqu.. `con` alto. Por otro lado, yo siempre había escuchado estrechamente acopladas que es una traducción más literal, aunque veo que alto/bajo es más técnico: https://es.wikipedia.org/wiki/Acoplamiento_(inform%C3%A1tica)

Reply:
Desde el punto de vista de estructura de datos, una lista y una cola difieren bastante. Por lo que sugiero mantener como cola ya que expresa mejor el modo de trabajo.

`it was placed in a queue` puede traducirse de dos formas distintas:
1. Colocar/poner en una cola
2. Encolar

```
... componente o alguna actualización estaba lista, se colocaba en la cola para el siguiente equipo...
```

Reply:
@electrocucaracha @raelga puedo colocar cola, solo que justo coloque fila para dar un contexto menos formal de computación. Porque en mi interpretación, no estamos hablando de estructura de datos, sino de una expresión relacionado a procesos largos. 
```
it went through so many developers, waiting in so many queues that it was difficult to trace the origin of the problem if the code didn’t work. 
```

Reply:
Sigue la conversación en el siguiente comentario de revisión.